src/xrGame/configs_dump_verifyer.cpp

```diff
@@ -197,20 +197,17 @@ bool const configs_verifyer::verify(u8* data, u32 data_size, string256& diff)
 
     m_orig_config_body.w_stringZ(add_str);
 
-    crypto::xr_sha256 tmp_sha_checksum;
-    tmp_sha_checksum.start_calculate(m_orig_config_body.pointer(), m_orig_config_body.tell());
-    while (!tmp_sha_checksum.continue_calculate())
-    {
-    };
+    crypto::xr_sha1::hash_t hash{};
```

Comment:
Given that 'crypto::xr_sha1::calculate()' is now a single self-contained function, you can return the hash from it without passing it by reference.

Reply:
Will do